[2023-10-20 23:28:10,661] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-08-26T00:00:00+00:00 [queued]>
[2023-10-20 23:28:10,677] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-08-26T00:00:00+00:00 [queued]>
[2023-10-20 23:28:10,677] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:28:10,678] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:28:10,679] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:28:10,712] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): tercera_tarea> on 2023-08-26 00:00:00+00:00
[2023-10-20 23:28:10,720] {standard_task_runner.py:52} INFO - Started process 296 to run task
[2023-10-20 23:28:10,725] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'tercera_tarea', 'scheduled__2023-08-26T00:00:00+00:00', '--job-id', '1188', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmprc0ts_cu', '--error-file', '/tmp/tmp4yilyzrw']
[2023-10-20 23:28:10,728] {standard_task_runner.py:80} INFO - Job 1188: Subtask tercera_tarea
[2023-10-20 23:28:10,868] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2023-08-26T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:28:11,011] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=tercera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-08-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-26T00:00:00+00:00
[2023-10-20 23:28:11,014] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:28:11,016] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2']
[2023-10-20 23:28:11,029] {subprocess.py:85} INFO - Output:
[2023-10-20 23:28:11,031] {subprocess.py:92} INFO - hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2
[2023-10-20 23:28:11,032] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:28:11,093] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=tercera_tarea, execution_date=20230826T000000, start_date=20231020T232810, end_date=20231020T232811
[2023-10-20 23:28:11,149] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:28:11,207] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
